css/shared/src/main/scala/io/udash/css/CssBase.scala

```diff
@@ -152,17 +153,21 @@ trait CssBase {
           ))(Env.empty)
         )
       case CssKeyframes(className, steps) =>
+        val kfs = Keyframes(
+          new ClassName(className),
+          steps.map { case (p, impl) =>
+            (Percentage(p): scalacss.internal.KeyframeSelector, StyleA(
+              new ClassName(className),
+              Vector.empty,
+              impl
+            ))
+          }
+        )
         renderer.apply(Vector(
-          Css.keyframes(Keyframes(
-            new ClassName(className),
-            steps.toSeq.map { case (p, impl) =>
-              (Percentage(p): scalacss.internal.KeyframeSelector, StyleA(
-                new ClassName(className),
-                Vector.empty,
-                impl
-              ))
-            }
-          ))(Env.empty)
+          CssEntry.Keyframes(
+            kfs.name,
+            ListMap(kfs.frames.map(s => (s._1, styleA(s._2)(Env.empty))):_*)
```

Comment:
1. Use pattern matching syntax for untupling. 
2. You can avoid copying the mapped sequence with breakOut and explicit type.
```scala
kfs.frames.map(s => (s._1, styleA(s._2)(Env.empty)))(collection.breakOut): ListMap[KeyframeSelector, StyleStream]
```